pkg/apps/strategy/rolling/rolling.go

```diff
@@ -125,6 +128,11 @@ func (s *RollingDeploymentStrategy) Deploy(from *kapi.ReplicationController, to
 	params := config.Spec.Strategy.RollingParams
 	updateAcceptor := s.getUpdateAcceptor(time.Duration(*params.TimeoutSeconds)*time.Second, config.Spec.MinReadySeconds)
 
+	recreateTimeout := time.Duration(appsapi.DefaultRollingTimeoutSeconds) * time.Second
```

Comment:
@tnozicka i wonder where we set the timeout for hooks before... i haven't seen the code anywhere, guess we rely on the deadline of the deployer pod to remove them, if that is the case, i will probably remove all timeout code.

Reply:
https://github.com/tnozicka/origin/blob/e050bcffadc6fa8fe12e9d306eb36c9aebe466c8/pkg/apps/strategy/support/lifecycle.go#L342-L346